.github/workflows/ci.yml

```diff
@@ -1,49 +1,28 @@
+name: CI
```

Comment:
I think there is something wrong here.

We need to keep the CI as is was before. (ok with names update but nothing else)